src/events/messageCreate/checkAvailability.js

```diff
@@ -0,0 +1,252 @@
+    },
+    {
+        'role':    'user',
+        'content': I free from 9-5 monday through friday. Message date 6/12/2024,
+    },
+    {
+        'role':    'assistant',
+        'content': {name: "rememberAvailability", arguments: "{"from":"09:00","to":"17:00","days":[true,true,true,true,true]}",},
+    },
+    `;
+
+/**
+ * Given a function object from open ai and a message, attempt to parse 
+ * @param {DiscordJS Message} message 
+ * @param {OpenAI Tool} calledFunction 
+ */
+const parseResults = (message, calledFunction) =>{
+    const action = {
+        'rememberUnavailability': () => rememberUnavailability(message, JSON.parse(calledFunction.arguments)),
+        'rememberAvailability':   () => rememberAvailability(message, JSON.parse(calledFunction.arguments)),
+        'unableToParse':          () => unableToParse(message),
+    };
+    action[calledFunction.name]();
+};
+
+// checks the message sent in the channel and sends it to openai to parse, then if possible saves the availability data
+module.exports = async (client, message) => {
+    try {
+        const availabilityChannel = await getAvailabilityChannel();
+
+        // only send a message if it's not from a bot and it's from the available channel
+        if (message.author.bot || !availabilityChannel || message.channelId !== availabilityChannel.id) {
+            return;
+        }
+        const date = new Date(message.createdTimestamp).toLocaleDateString();
+        const prompt = stripIndents`Based on the attached message and examples, 
+        use one of the provided tools to parse unavailability using 'rememberUnavailability' or availability using 'rememberAvailability'. 
+        If either don't work call 'unableToParse' from the tools.The message date is ${date}.`;
+        const response = await openAiClient.chat.completions.create({
+            model:    'gpt-3.5-turbo',
+            messages: [
+                {
+                    'role':    'system',
+                    'content': prompt,
+                },
+                {
+                    'role':    'system',
+                    'content': stripIndents`These are some example input and outputs to mimic. The dates don't align to the current message date so make sure to parse with the current message date. ${examples}`
+                },
+                {
+                    'role':    'system',
+                    'content': `Avoid calling both 'rememberUnavailability' and 'rememberAvailability' unless the user's wording makes it clear and necessary. `
+                },
+                {
+                    'role':    'user',
+                    'content': stripIndents`${message.content}`,
+                }
```

Comment:
Should have only one system prompt. Don't think it's made to accommodate multiple. So easiest way to do what you have here is combine all the system prompts into one, then keep your user message.